tests/testthat/test-transformation.R

```diff
@@ -0,0 +1,24 @@
+library(shapr)
```

Comment:
You should have this file in a separate PR. Don't know if this was intentional or not.